deployment/network-operator/README.md

```diff
@@ -218,15 +218,62 @@ helm upgrade -n network-operator  network-operator mellanox/network-operator --v
 
 >__NOTE__: `--devel` option required if you want to use the beta release
 
-### Return PODs to the drained nodes
 
->__NOTE__: this operation required only if **containerized OFED** is in use and if drain command
-was applied before to remove PODs from the cluster nodes
+### Restart PODs with containerized OFED driver
+
+>__NOTE__: this operation required only if **containerized OFED** is in use
+
+Helm upgrade command will upgrade DaemonSet of the OFED driver to point to the new driver version,
+but will not remove or restart PODs with the driver on the nodes.
```

Comment:
can you clarify why restart of pods wont happen ? e.g due to the daemonset update strategy, the daemonset will not remove or restart PODs with the driver on nodes.

atm it reads (to me) as if helm is doing it.

Reply:
I updated this section in the doc